src/main.ts

```diff
@@ -63,22 +64,21 @@ fetchConfig()
     // const authzSerice = new AuthorizationService() // FUTURE
     // await authzSerice.getAuthorizations('NR123456') // FUTURE
     */
-    if (featureFlags.getFlag('bcrs-create-ui-enabled')) {
```

Comment:
This was a recent fix that you're undoing. Please revert this change.

Reply:
poor rebase on my part, corrected in latest commit